pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/PutFormAnswersRoute.java

```diff
@@ -150,10 +152,27 @@ public Object handle(Request request, Response response) {
                     // FIXME: address doesn't get saved until this PUT call finishes, so we couldn't check address here.
                     // checkAddressRequirements(handle, userGuid, form);
 
+                    User participantUser = instanceSummary.getParticipantUser();
+                    User operatorUser = participantUser;
+                    if (!userGuid.equals(operatorGuid)) {
+                        operatorUser = handle.attach(UserDao.class).findUserByGuid(operatorGuid)
+                                .orElseThrow(() -> new DDPException("Could not find operator with guid " + operatorGuid));
+                    }
+
+                    var instanceStatusDao = handle.attach(ActivityInstanceStatusDao.class);
+
                     List<ActivityValidationFailure> validationFailures = actValidationService.validate(
                             handle, interpreter, userGuid, operatorGuid, instanceGuid, form.getCreatedAtMillis(),
                             form.getActivityId(), preferredUserLangDto.getId());
                     if (!validationFailures.isEmpty()) {
+                        //check if study has errorPresentStatus enabled
+                        StudyDto studyDto = new JdbiUmbrellaStudyCached(handle).findByStudyGuid(studyGuid);
+                        if (studyDto.isErrorPresentStatusEnabled()) {
+                            instanceStatusDao.updateOrInsertStatus(instanceDto, InstanceStatusType.ERROR_PRESENT,
+                                    Instant.now().toEpochMilli(), operatorUser, participantUser);
+                            return new PutAnswersResponse(WorkflowResponse.unknown());
```

Comment:
if validations fail and ERROR_PRESENT status is enabled for this study, update activity to this status and return.
angular will goto dashboard.